swig/build-utils.shinc

```diff
@@ -253,6 +274,12 @@ build_libraries() {
     mv swig/bin/release/${os_build}/${os_arch}/${SHARED_LIB} swig/bin/release/${os_build}/${os_arch}/${RELEASE_SHARED_LIB}
     cp swig/bin/release/${os_build}/${os_arch}/${RELEASE_SHARED_LIB} .
   fi
+  if [[ $os_build == "ios" ]]; then
```

Comment:
please DRY (Don't repeat yourself), just add an OR condition on the "macosx" if, the code block you added is verbatim the same as the "maxosx" one.